### ISSUE TITLE
change the license with the Hippocratic license 

### ISSUE THREAD

**Issue by baldarn** [getify/youperiod.app#15]
I would suggest to change the license with this, that will prevent the unethical usage of the code of the app

https://firstdonoharm.dev/

**Comment by getify** [comment]
At a glance, I can certainly see why this would be attractive for this project, and I like the goals of that license.

I am however a bit reluctant to switch quickly to it. I have spent a long time in open-source, and strongly feel that well known and deeply studied licenses are important to standardize on. I have resisted the urge on many occasions to try to modify/author my own license text to try to socially persuade users of my code to do "the right thing". But in the end, and after lots of careful thought and research, I kept landing back at MIT.

It doesn't prevent someone from using the code for bad purposes. I wish it did. But then the license can't be considered truly open-source (i.e., it's "non-free") if it includes such language. There's been many famous examples of this tension over the years, including [Doug Crockford's license (for JSON, etc)](https://www.change.org/p/douglas-crockford-remove-the-not-evil-clause-from-your-license-because-it-is-evil-itself), which upset a lot of people.

So anyway, this is something we might consider, but I'm not ready to give it a blessing yet. I'd like more information on other projects using it, what open-source lawyers say about it, how this license might make it harder for our project's code to be used/audited, etc.

In short: I want to tread carefully before deviating from MIT, which I've known and used and trusted for ~15 years of open-source work.

**Comment by baldarn** [comment]
Good points!

Here is a list of projects currently using the license
https://firstdonoharm.dev/adopters/

And here a list of FAQs 
https://firstdonoharm.dev/learn/

I think that freedom is having the ability to do what has to be done, not the ability of doing anything.
